pkg/util/utils.go

```diff
@@ -32,6 +32,15 @@ func ListContainsSubString(list []string, s string) bool {
 	return false
 }
 
+func StringContains(s string, list []string) bool {
```

Comment:
This name should be something like `StringContainsAnyFromList` to differentiate from strings.Contains()

Reply:
done